java/compaction/compaction-job-creation/src/main/java/sleeper/compaction/job/creation/CreateCompactionJobs.java

```diff
@@ -114,38 +117,57 @@ public enum Mode {
     }
 
     public void createJobs(TableProperties table) throws StateStoreException, IOException, ObjectFactoryException {
-        LOGGER.info("Performing pre-splits on files in {}", table.getStatus());
+        LOGGER.info("Creating compaction jobs for table {}", table.getStatus());
```

Comment:
This seems to duplicate the log message at the start of createJobsForTable. That one's at debug right now, maybe that should be info?